mtags/src/main/scala-2/scala/meta/internal/pc/AutoImportsProvider.scala

```diff
@@ -52,13 +52,21 @@ final class AutoImportsProvider(
       case _ => false
     }
 
+    def correctInTreeContext(sym: Symbol) = lastVisitedParentTrees match {
+      case (_: Ident) :: (sel: Select) :: _ =>
+        sym.info.members.exists(_.name == sel.name)
+      case (_: Ident) :: (_: Apply) :: _ =>
+        sym.info.members.exists(_.name == nme.apply)
```

Comment:
How will this work if `sym` is a function? Just making sure it won't throw.

Reply:
Do you have a case in mind? We wouldn't normally offer auto imports for methods so I assumed it's not an issue 🤔

Reply:
<img width="376" alt="Screenshot 2025-03-06 at 18 37 51" src="https://github.com/user-attachments/assets/fc0224e0-1b88-4091-af55-001307fac946" />
?